internal/db/db_blueprints.go

```diff
@@ -26,6 +26,11 @@ const (
 		INSERT INTO blueprint_versions(id, blueprint_id, version, body)
 		VALUES($1, $2, $3, $4)`
 
+	sqlLatestBlueprintVersion = `
+		SELECT MAX(blueprint_versions.version)
+		FROM blueprint_versions INNER JOIN blueprints ON blueprint_versions.blueprint_id = blueprints.id
+		WHERE blueprints.org_id = $1 AND blueprints.id = $2`
+
```

Comment:
Have you tested with multiple versions? I believe this query will return multiple records if you have multiple versions which `QueryRow` will silently ignore, yet, all those records will be transferred from DB to the backend for no reason. I suggest to add `LIMIT 1` here.

Reply:
I test it with multiple versions, but might be just accident it works in the tests.

IMO `MAX()` without grouping selects only single row tho, or am I missing something?

Reply:
Yeah that is right there is no group so it will return single record. What I was thinking, okay.